source/math/FloatVectorList.ooc

```diff
@@ -181,16 +186,10 @@ FloatVectorList: class extends VectorList<Float> {
 		max != 0 ? (this / max) : this copy()
 	}
 	getOnes: static func (count: Float) -> This {
-		result := This new(count)
-		for (i in 0 .. count)
-			result add(1.0f)
-		result
+		This new(count, 1.0f)
 	}
 	getZeros: static func (count: Float) -> This {
-		result := This new(count)
-		for (i in 0 .. count)
-			result add(0.0f)
-		result
+		This new(count, 0.0f)
```

Comment:
And then this function will do exactly what the `createDefault` function in `FloatComplexVectorList`, which you are currently removing, does. So maybe you shouldn't remove that function there, but rename it to `getZeroes` and save a few lines of code by calling 
```ooc
This getZeros(count)
```
instead of 
```
result := This new(input count)
result _count = input count
```
in, what, 4 places.